tabbycat/privateurls/views.py

```diff
@@ -149,7 +152,7 @@ def post(self, request: 'HttpRequest', *args, **kwargs) -> 'HttpResponseRedirect
 
 class EmailRandomisedUrlsView(RoleColumnMixin, TournamentTemplateEmailCreateView):
     page_subtitle = _("Private URLs")
-
+    edit_permission = Permission.SEND_PRIVATE_URLS
```

Comment:
Should also add a view permission here, maybe a new one as this also has email addresses, so more personal details.